server/api/v1/controllers/userController.js

```diff
@@ -0,0 +1,139 @@
+// server/src/api/v1/controllers/userController.js
+import User from '../models/User.js';

```

Comment:
check the file name that you are importing from and correct it

Reply:
I did change it